spec/spec_tests/crud_spec.rb

```diff
@@ -6,27 +6,25 @@
 
     spec = Mongo::CRUD::Spec.new(file)
 
-    before do
-      unless spec.server_version_satisfied?(authorized_client)
-        skip 'Version requirement not satisfied'
-      end
-    end
-
     context(spec.description) do
 
       spec.tests.each do |test|
 
         context(test.description) do
 
           before(:each) do
+            unless spec.server_version_satisfied?(authorized_client)
```

Comment:
When testing locally, this block skipped _all_ of the tests even when not all of them entered the `unless` block; moving this block here seemed to fix it.